browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -157,63 +157,6 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 		}, this);
 	},
 
-	_onInvalidateTilesMsg: function (textMsg) {
-		var command = app.socket.parseServerCmd(textMsg);
-		if (command.x === undefined || command.y === undefined || command.part === undefined) {
-			var strTwips = textMsg.match(/\d+/g);
-			command.x = parseInt(strTwips[0]);
-			command.y = parseInt(strTwips[1]);
-			command.width = parseInt(strTwips[2]);
-			command.height = parseInt(strTwips[3]);
-			command.part = this._selectedPart;
-		}
-
-		if (isNaN(command.mode))
-			command.mode = this._selectedMode;
-
-		var topLeftTwips = new L.Point(command.x, command.y);
-		var offset = new L.Point(command.width, command.height);
-		var bottomRightTwips = topLeftTwips.add(offset);
-		if (this._debug.tileInvalidationsOn && command.part === this._selectedPart) {
-			this._debug.addTileInvalidationRectangle(topLeftTwips, bottomRightTwips, textMsg);
-		}
-
-		var invalidBounds = new L.Bounds(topLeftTwips, bottomRightTwips);
-		var visibleArea, visiblePaneAreas;
-		if (this._splitPanesContext) {
-			visiblePaneAreas = this._splitPanesContext.getTwipsBoundList();
-		}
-		else {
-			var visibleTopLeft = this._latLngToTwips(this._map.getBounds().getNorthWest());
-			var visibleBottomRight = this._latLngToTwips(this._map.getBounds().getSouthEast());
-			visibleArea = new L.Bounds(visibleTopLeft, visibleBottomRight);
-		}
-
-		var needsNewTiles = false;
-		for (var key in TileManager.tiles) {
-			var coords = TileManager.get(key).coords;
-			var bounds = this._coordsToTileBounds(coords);
-			if (coords.part === command.part && coords.mode === command.mode &&
```

Comment:
A) in this original block if we remove the intersectsVisible use for a moment to make this more clear, this this originally reads:

if (coords.part === command.part && coords.mode === command.mode &&         invalidBounds.intersects(bounds)) {
        TileManager.invalidateTile(key, command.wireId);
}

so the TileManager.invalidateTile depends on the invalidBounds.intersects(bounds), not the intersectsVisible